src/pattern-library/components/LibraryNext.js

```diff
@@ -57,6 +57,35 @@ function ChangelogItem({ status, children }) {
   );
 }
 
+/**
+ *
+ * @param {object} props
+ *   @param {string} props.content
+ *   @param {'sm'|'md'|'lg'} [props.size]
```

Comment:
Good feedback. I'll rethink.

Reply:
@robertknight 

Interesting food for thought here. In terms of "how the other guys do it", it's split (this survey was specific to `Button` components so is not exhaustive for sizing across the entire library):

* MUI: `small`, `medium`, `large`
* Mantine: `sm`, `md`, `lg`
* Ant: `large`, `small` (oddly, no value for the default size. Weird)
* NextUI: `sm`, `md`, `lg`, `xl`
* Reactstrap: `sm`, `lg`
* Chakra: `sm`, `md`, `lg`, `xl`

I almost just went and switched to `small`, `medium` and `large`, especially since MUI does it and they are most closely aligned to our overall approach. But I was swayed by two factors:

* Ergonomics: It's just a bit faster to use (standardized and well-recognized) abbreviations
* More relevantly, systematized-ness (oy, not a word): The abbreviations are easier to reason about if we ever got beyond `sm`, `md`, `lg`. It's conventionally obvious that the thing after `lg` would be `xl`. But it's less conventionally obvious what the thing after `large` is. I don't know whether we'd ever need sizes beyond the these three in real life, but in the end I have a preference for `sm`, `md`, `lg` because it's more systematized.